apps/core/models.py

```diff
@@ -131,7 +131,28 @@ def __str__(self):
         return self.name
 
 
-class OmicsUnit(models.Model):
+class UUIDModelMixin(object):
+
+    def __str__(self):
+
+        print(self.__class__.__name__)
+        if self.__class__.__name__ == "OmicsUnit":
```

Comment:
You should override the `__str__` method of the `OmicsUnit` model inheriting from the `UUIDModelMixin`. This is more explicit.